dadi/lib/handlers/image.js

```diff
@@ -1020,6 +1026,23 @@ ImageHandler.prototype.processGif = function (buffer) {
   })
 }
 
+/**
+ * Transcodes an input buffer to a progressive JPEG
+ *
+ * @param {Buffer} buffer - a Buffer extracted from the main image
+ * processor after applying image manipulations
+ * @returns {Buffer} a progressive JPEG encoded buffer
+ */
+ImageHandler.prototype.processJpeg = function (buffer) {
+  return imagemin.buffer(buffer, {
+    plugins: [
+      imageminJpegtran({progressive: true})
+    ]
+  }).then(outputBuffer => {
```

Comment:
Do we need this `then`?

Reply:
Is there a header we can inspect to tell if a client supports progressive jpg? @abovedave submitted the issue so am deferring to him.

Reply:
Got spot, I'll remove that